.happy/terraform/envs/rdev/variables.tf

```diff
@@ -0,0 +1,42 @@
+variable aws_account_id {
+  type        = string
+  description = "AWS account ID to apply changes to"
+  default     = ""
+}
+
+variable aws_role {
+  type        = string
+  description = "Name of the AWS role to assume to apply changes"
+  default     = ""
+}
+
+variable image_tag {
+   type        = string
+  description = "Please provide an image tag"
+}
+
+variable priority {
+  type        = number
+  description = "Listener rule priority number within the given listener"
+}
```

Comment:
what are these listeners for and why do they need to be assigned priorities?

Reply:
These listeners are routing rules for Amazon Application Load Balancers. Our configuration looks for exact host name matches, so the priority values aren't especially important to us, the AWS API requires that these values are set and unique.